cobc/ChangeLog

```diff
@@ -3,6 +3,8 @@
 
 	* codegen.c, cconv.h, cconv.c: extract EBCDIC & ASCII conversion tables
 	  to a dedicated module
+	* flag.def: add -febcdic-table flag to customize EBCDIC translation;
+	  drop -falernate-ebcdic (replaced by -febcdic-table=restricted-gc)
```

Comment:
As this is (likely) in use we don't drop that. Th following in cobc.c  (around line 610) should work (possibly with some casting):

```c
	/* alias for backwards-compatibility, removed with 4.x:
	   possibly add to help.c, possibly only output there
	   if verbose mode is used:
	  "alternate-ebcdic use restricted ASCII to EBCDIC translate" */
	{"falternate-ebcdic",		CB_NO_ARG, &cb_ebcdic_table, CB_EBCDIC_RESTRICTED_GC},
	{"fno-alternate-ebcdic",	CB_NO_ARG, &cb_ebcdic_table, CB_EBCDIC_DEFAULT},
```